packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -111,188 +78,139 @@ const propTypes = {
     'accept',
     'reject',
   ]),
+  /**
+   * Whether or not to show notification-dialog.
+   */
+  isOpen: PropTypes.bool,
```

Comment:
I'd like to propose dropping the isOpen prop. If consumers don't want to see a NotificationDialog, they shouldn't render one.

Reply:
I started to make this change and this results in
```js
        <NotificationDialog
          variant={variant}
          isOpen={isOpen}
          dialogTitle="The title relates directly to the choices."
          startMessage="...."
          acceptAction={{
            text: 'Confirm',
            onClick: handleCloseModal,
          }}
          rejectAction={{
            text: 'Close',
            onClick: handleCloseModal,
          }}
        />
```
to
```js
      {isOpen && (
        <NotificationDialog
          variant={variant}
          dialogTitle="The title relates directly to the choices."
          startMessage="...."
          acceptAction={{
            text: 'Confirm',
            onClick: handleCloseModal,
          }}
          rejectAction={{
            text: 'Close',
            onClick: handleCloseModal,
          }}
        />
      )}

```

I actually think `isOpen` may be cleaner but I'm okay with the change if others want it go as well.

Reply:
I see what you mean, though I prefer it gone because it brings us into alignment with logical rendering of all our components. We don't have `isVisible` props on our List or on our Button, we just don't render them if we don't want to see them.

The latter implementation also has the benefit of not instantiating the component just for the component to null itself out based on a prop. We don't have to create the new accept/rejectAction objects, don't have to mount the component, don't have to execute and return early from effects. For the NotificationDialog specifically, not of this has a huge impact, but it will impact some of our other modal implementations more harshly, and I'd like to be consistent if we can.

Reply:
You've convinced me.! Now to update my jest logic 👀 😄

Reply:
+1 to tylers suggestion

Reply:
It will align with decisions we've made with other components such as overlay etc...

Reply:
remove `isOpen` https://github.com/cerner/terra-framework/pull/1105/commits/440c28411da12f4c7b6d14c932c0437e8a289e63